parser/importer_pytesseract.py

```diff
@@ -0,0 +1,170 @@
+# Python3
+# coding: utf-8
+#
+# Commit note of Daniela Grothe
+# This version uses pytesseract.
+# Only tested on Windows 8 with Python 3.8
+# and ImageMagick 6.8.8-10 Q8 x64
+# Seems as if either Win8-cmd or ImageMagick didn't want 
+# as many high-commas (didn't find my files, error msg such as:
+# "convert.exe: unable to open image `\Users\daniela\..."
+# ) so I removed some of them in the convert commands.
+# 
+#
+# Copyright 2015-2018
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import os
+
+from PIL import Image
+
+BASE_PATH = os.getcwd()
+INPUT_FOLDER = os.path.join(BASE_PATH, "data\\img")
+TMP_FOLDER = os.path.join(BASE_PATH, "data\\tmp")
+OUTPUT_FOLDER = os.path.join(BASE_PATH, "data\\txt")
+
+# --- pytesseract prep
+try:
+    from PIL import Image
+except ImportError:
+    import Image
+import pytesseract
+
+pytesseract.pytesseract.tesseract_cmd = r'C:\Program Files\Tesseract-OCR\tesseract.exe'
```

Comment:
That's nice, but it only works for other users if they use the same path.

Another way would be to add the tesseract executable in your `PATH` environment variable.
I haven't tried it, but this should work:
```
setx path "%path%;C:\Program Files\Tesseract-OCR\"
```
Then you could remove the `pytesseract.pytesseract.tesseract_cmd` line and it should work for others, too. 😉